integration-cli/docker_cli_run_unix_test.go

```diff
@@ -178,26 +178,26 @@ func TestRunContainerWithCgroupParentAbsPath(t *testing.T) {
 func TestRunDeviceDirectory(t *testing.T) {
 	testRequires(t, NativeExecDriver)
 	defer deleteAllContainers()
-	cmd := exec.Command(dockerBinary, "run", "--device", "/dev/snd:/dev/snd", "busybox", "sh", "-c", "ls /dev/snd/")
+	cmd := exec.Command(dockerBinary, "run", "--device", "/dev/:/dev/", "busybox", "sh", "-c", "ls /dev/")
```

Comment:
My dev-vm doesn't. However when I checkout master and rerun the tests they worked, so I didn't think it was my environment. If that's the issue I can just revert the test and it should be fine.